packages/donations-app/devServer/controller/project.js

```diff
@@ -175,9 +176,20 @@ module.exports = function (app) {
         await new ProjectHistory({
           project: savedProject,
           outcomes: req.body.outcomes,
+          validators: req.body.validators,
           changedBy: req.user._id
         }).save();
 
+        // Setting validator
+        if (AccessControl.isSuperadmin(req.user)
+            && req.body.validators
+            && req.body.validators.length > 0)
+        {
+          let user = await User.findById(req.body.validators[0]);
+          user.validator.push(savedProject._id);
```

Comment:
Do we handle the update? 
Are we doing anything to the previous validator (like cleaning the project from the previous user validators list)?